src/js/components/Layer/StyledLayer.js

```diff
@@ -210,27 +219,27 @@ const POSITIONS = {
       bottom: ${margin.bottom};
       left: 50%;
       transform: translateX(-50%);
-      animation: ${KEYFRAMES.center.vertical} 0.2s ease-in-out forwards;
+      animation: ${props => getKeyframes(props.animate, "center", "vertical")}
```

Comment:
I think we need to move `animation` inside `getKeyFrames`, I think know would be better to even rename this function to `getAnimationStyle`.

from

```
animation: ${props => getKeyframes(props.animate, "center", "vertical")}
```

to

```
${props => getAnimationStyle(props.animate, "center", "vertical")}
```

inside animation style we could do:

```
if (!animate) {
  return '';
}
```